common/tools/dev-tool/src/util/testProxyUtils.ts

```diff
@@ -14,7 +14,7 @@ const log = createPrinter("test-proxy");
 const CONTAINER_NAME = "js-azsdk-test-proxy";
 
 export async function startProxyTool(): Promise<void> {
-  log.info(`Attempting to start test proxy at http://localhost:5000 & https://localhost:5001.\n`);
```

Comment:
Should we add a variable for the HTTPS port while we're at it?

Reply:
I was thinking we can get to HTTPS when we get to fixing the issues with certificates and the recorder.

Reply:
I feel like it should be easy enough to add the environment variable in this PR while we're at it, it touches all the same places.